docs/src/meshes/MeshDataMeshes.md

```diff
@@ -0,0 +1,70 @@
+* `rd.Vf`: interpolation matrices from values at nodal interpolation points to face quadrature points
+* `rd.wf`: a vector containing face quadrature weights on the reference element
+* `rd.M`: the quadrature-based mass matrix, computed via `rd.Vq' * diagm(rd.wq) * rd.Vq`.
+* `rd.Pq`: a quadrature-based ``L^2`` projection matrix `rd.Pq = rd.M \ rd.Vq' * diagm(rd.wq)` which maps between values at quadrature points and values at nodal points. 
+* `Dr, Ds, Dt` matrices are nodal differentiation matrices with respect to the ``r,s,t`` coordinates, e.g., `Dr*f.(r,s)` approximates the derivative of ``f(r,s)`` at nodal points. 
+
+Quantities in `md::MeshData`: 
+* `md.xyz` is a tuple of matrices `md.x`, `md.y`, `md.z`, where column `e` contains coordinates of physical interpolation points. 
+* `md.xyzq` is a tuple of matrices `md.xq`, `md.yq`, `md.zq`, where column `e` contains coordinates of physical quadrature points. 
+* `md.rxJ, md.sxJ, ...` are matrices where column `e` contains values of ``J\frac{\partial r}{\partial x}``, ``J\frac{\partial s}{\partial x}``, etc. at nodal interpolation points on the element `e`.
+* `md.J` is a matrix where column `e` contains values of ``J`` at nodal interpolation points.
+* `md.sJ` is a matrix where column `e` contains values of the face Jacobian (e.g., determinant of the geometric mapping between a physical face and a reference face) at face quadrature points.
+* `md.nxJ, md.nyJ, ...` are matrices where column `e` contains values of components of the unit normal scaled by the face Jacobian `md.sJ` at face quadrature points.
+
+For more details, please see the [StartUpDG.jl docs](https://jlchan.github.io/StartUpDG.jl/dev/). 
+## Special options
+
+Trixi solvers on simplicial meshes use `DG` solver types with the basis field set as a `RefElemData` type. By default, `RefElemData(Tri(), N)` will generate data for a degree ``N`` polynomial approximation on a triangle. There are also several parameters which can be tweaked:
+
+* `RefElemData(Tri(), N, quad_rule_vol = quad_nodes(Tri(), Nq))` will substitute in a volume quadrature rule of degree `Nq` instead of the default (which is a quadrature rule of degree `N`). Here, a degree `Nq` rule will be exact for at least degree `2*Nq` integrands (such that the mass matrix is integrated exactly).
+* `RefElemData(Tri(), N, quad_rule_face = quad_nodes(Line(), Nq))` will use a face quadrature rule of degree `Nq` rather than the default. 
+* `RefElemData(Tri(), SBP(), N)` will generate a reference element corresponding to a multi-dimensional SBP discretization. Types of SBP discretizations available include `SBP{Kubatko{LobattoFaceNodes}}()` (the default choice), `SBP{Kubatko{LegendreFaceNodes}}()`, and `SBP{Hicken}()`. Descriptions of each can be found in the [StartUpDG.jl docs](https://jlchan.github.io/StartUpDG.jl/dev/RefElemData/#RefElemData-based-on-SBP-finite-differences). Trixi will also specialize based certain parts of the solver based on the `SBP` type. 
```

Comment:
Is there a paper for the methods of Kubatko? I could only find paper references for the other two options at https://jlchan.github.io/StartUpDG.jl/dev/RefElemData/#RefElemData-based-on-SBP-finite-differences.

Reply:
Unfortunately no - he just sent the nodes over private communication. The 'Kubatko{LegendreFaceNodes}' correspond to the SBP nodes of Chen and Shu 2017 though. Do you think we should cite that for the Kubatko nodes, with a note that the Kubatko rules are more general?

Reply:
That's a good idea :+1: